docs/index.html

```diff
@@ -1 +1,35 @@
-Test
+<!DOCTYPE html>
+<html>
```

Comment:
I usually do a `lang`-Attribute on the html-tag:

* [why?](https://www.w3.org/International/questions/qa-lang-why)
* [how?](https://www.w3schools.com/tags/ref_language_codes.asp)

Reply:
Yep, thanks, added lang="en"